NEWS.md

```diff
@@ -1,76 +1,113 @@
-3.19.1
+3.20.1
 ======
 
 ## Features
 
- * HTTP load balancer (#2347)
- * Slack destination (#2451)
- * Add Cisco Catalyst formatted triplets support to cisco-parser() (#2394)
- * Add RFC5424 syslog support to the system() source (FreeBSD 12.0 support) (#2430)
- * Add network `interface()` option to network sources (#2389)
- * Add so-reuseport() to network drivers (#2379)
- * Enable supporting HTTP redirects (#2136)
+ * Add persist-tool (#2511)
+ * Collectd destination (#1433)
+ * Config reload status feedback (#2367)
+ * Example-msg-generator: num option (#2565)
+ * Json list support (#2536)
+ * Netskope parser (#2541)
+ * Python-destination: send can return worker_insert_result_t, flush support (#2487)
+ * Python: internal() source exposed via syslogng.Logger (#2505)
+ * Websense parser (#2471)
+ * Xml/geoip2: make prefix optional (#2538)
+ * Xml-parser: add list support (#2544)
```

Comment:
IMHO, Netskope, Collectd, Websense, persist-tool, Config reload status feedback should be on the top, they are more independent features than for example python-related improvements.

Reply:
I used `sort` to order them, but sure I can reorder based on other criteria (but that is going to bias).

Reply:
It's not a must, just wanted to leave a note :)